www/apps/book/app/page.mdx

```diff
@@ -8,14 +8,15 @@ export const metadata = {
 
 # {pageNumber} Introduction
 
-Medusa is a digital commerce platform with a built-in framework for customizations and integrations.
+Medusa is a digital commerce platform with a built-in framework for customization.
 
-Medusa's framework includes two types of tools:
+Medusa is ships with three main tools:
 
-1. Modules that provide commerce functionalities, such as Product, Order, and Cart modules.
-2. Customization tools to build your business features, such as custom data models, business logic, flows, and endpoints.
+1. A suite of commerce modules that provide core commerce functionalities, such as tracking inventory, calculating cart totals, accepting payments, managing orders, and much more.
+2. A framework to build custom functionality that is specific to your business, product, or industry. This includes tools for introducing custom API endpoints, business logic, and data models; building workflows and automations; and integrating with third-party services.
+3. A customizable admin dashboard for merchants to configure and operate their store.
 
-With these tools, you save time you would spend with other platforms on maintaining and setting up your customizations. Instead, you rapidly build important features for your business.
+When you start Medusa the first time you get a fully fledged commerce platform with all the features you need to get off the ground. But unlike other platforms, Medusa is built with customization in mind. This avoids the need to build hacky workarounds that are difficult to maintain and scale, and instead allows you to focus on building features that move the needle.
```

Comment:
```suggestion
When you start Medusa the first time you get a fully fledged commerce platform with all the features you need to get off the ground.

But, unlike other platforms, Medusa is built with customization in mind. You don't need to build hacky workarounds that are difficult to maintain and scale. Instead, you can focus on building features that move the needle.
```

Not a fan of " you can focus on building features that move the needle." as it feels vague. What do you think of something like " you can focus on building features that brings your business's vision to life"?